client/src/views/Environment.tsx

```diff
@@ -32,38 +34,51 @@ function Environment() {
     if (!environment) return;
     const list = await okteto.endpoints(environment.file, environment.contextName);
     setEndpoints(list);
+
```

Comment:
Btw, if you use useEffect and interpret the value changes of the status you don't need to use useInterval.